.github/workflows/test-runner-integration.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Run Workspace Manager Integration Tests
+
+on:
+  push:
+    paths-ignore:
+      - 'README.md'
+      - 'local-dev/**'
+
+jobs:
+  test-runner-perf:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up AdoptOpenJDK 11
+        uses: joschi/setup-jdk@v2
+        with:
+          java-version: 11
+      - name: Get Vault token
+        id: vault-token-step
+        env:
+          VAULT_ADDR: https://clotho.broadinstitute.org:8200
+        run: |
+          VAULT_TOKEN=$(docker run --rm --cap-add IPC_LOCK \
+            -e "VAULT_ADDR=${VAULT_ADDR}" \
+            vault:1.1.0 \
+            vault write -field token \
+              auth/approle/login role_id=${{ secrets.VAULT_APPROLE_ROLE_ID }} \
+              secret_id=${{ secrets.VAULT_APPROLE_SECRET_ID }})
+          echo ::set-output name=vault-token::$VAULT_TOKEN
+          echo ::add-mask::$VAULT_TOKEN
+      - name: "Configure Test Runner"
+        run: |
+          cd ${GITHUB_WORKSPACE}/${workingDir}/workspace-manager-clienttests
+          chmod +x render-config.sh
+          ./render-config.sh ${{ steps.vault-token-step.outputs.vault-token }}
+          chmod +r rendered
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            .gradle/caches
+            .gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle') }}
+          restore-keys: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}
+      - name: Run GetWorkspace tests using test runner library
+        run: |
+          cd ${GITHUB_WORKSPACE}/${workingDir}/workspace-manager-client
+          ../gradlew clean
+          ../gradlew build
+          cd ${GITHUB_WORKSPACE}/${workingDir}/workspace-manager-clienttests
+          ./gradlew runTest --args="configs/integration/BasicAuthenticated.json /tmp/TestRunnerResults"
```

Comment:
nit: newline at EOF here and below

Reply:
It seems odd to have  to call gradlew three times in succession. Could we not have one job that cleans, builds, and runs the tests? Or is this because we're in a different gradle project for the tests?

Reply:
Should the client tests be their own independent project? Or are they a subproject? If the latter, they should not have their own gradle invocation. If the former, why?

Reply:
The first two `gradlew` operations have been merged into a single `gradlew` as suggested by @zarsky-broad. Right now it's just `./gradlew :workspace-manager-client:build`, or we can add back the `clean` task to the build process `./gradlew :workspace-manager-client:clean :workspace-manager-client:build`. This step builds a local client jar. 

Developers can specify the location of the local client jar through either a `gradle` property `workspacemanagerclientjar` or `env` `ORG_GRADLE_PROJECT_workspacemanagerclientjar` as described in the `README`.

The client tests were set up as independent project with their own `gradle`, avoiding having to make the decision to touch the main project `build.gradle` and `settings.gradle`. So far, this set up is sufficient. Team should decide if they want it to be.a subproject managed by the main `build.gradle`.